.github/dependabot.yaml

```diff
@@ -1033,20 +1033,6 @@ updates:
       - dependency-name: "amplify_lints"
       - dependency-name: "smithy_aws"
       - dependency-name: "aws_signature_v4"
-  - package-ecosystem: "pub"
-    directory: "packages/smithy/goldens/lib/custom"
```

Comment:
`custom` has smithy tests I wrote by hand. The `v1` / `v2` split is meant to track differences in Smithy IDL versions since the AWS-vended Smithy models were (in the past) not fully migrated to V2. I think they've all been migrated at this point, so we can probably remove all v1 soon.

However, for the handwritten tests, it's not necessary to test V1 since many of the tests are specifically testing V2 features.